src/modeler/AutoRest.Swagger/OperationBuilder.cs

```diff
@@ -249,6 +249,10 @@ private List<Stack<IModelType>> BuildResponses(Method method, CompositeType head
                             methodName,
                             response.Key));
                     }
+                    foreach (var key in _operation.Responses[response.Key].Extensions.Keys)
```

Comment:
I'd rather copy `_operation.Responses[response.Key].Extensions` to `method.Responses[response.Key.ToHttpStatusCode()].Extensions` directly (so yes, make the property settable, it's mutable anyways).